Assignment4/Formil-b190422/task07.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+"""Task07.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/17-APxnStNCFdK3NgHdNabdyMME_gX_ay
+
+**Task 07: Querying RDF(s)**
+"""
+
+#!pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+from rdflib.plugins.sparql import prepareQuery
+
+"""**TASK 7.1: List all subclasses of "Person" with RDFLib and SPARQL**"""
+print("TASK 7.1")
+from rdflib.plugins.sparql import prepareQuery
+ns = Namespace("http://somewhere#")
+rdfs = Namespace("http://www.w3.org/2000/01/rdf-schema#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+print("Printeamos los resultados obtenidos con rdfslib")
+for s,p,o in g.triples((None,rdfs.subClassOf,ns.Person)):
+  print(s)
+# Visualize the results
+
+q1 = prepareQuery('''
+  SELECT ?Subject WHERE { 
+    ?Subject rdfs:subClassOf ns:Person
```

Comment:
missing all other subclasses of person. Also, you are copying this from last year. This is not this year's assignment. I will stop my comments here.